src/components/molecules/idCopyButton.tsx

```diff
@@ -31,7 +37,13 @@ export const IdCopyButton = ({
 			<Button className={buttonClassName} tabIndex={-1} variant={variant}>
 				{displayFullLength ? id : idStr}
 			</Button>
-			<CopyButton className="mb-0.5" size="sm" successMessage={successMessage} tabIndex={0} text={id} />
+			<CopyButton
```

Comment:
We need to reduce the IdCopyButton icon size everywhere in the system, so we don't need the size property, and I think we also don't need the `iconClassName` property, if it should be the same everywhere.